src/script/conversation/linkPreviews/helpers.test.ts

```diff
@@ -100,6 +108,20 @@ describe('getFirstLinkWithOffset', () => {
     expect(Link.url).toEqual('wire.com');
   });
 
+  it('should return the correct link and offset for a single link preceded by a code block', () => {
+    const Link = getFirstLinkWithOffset('```\ntrap.com\n```\nwire.com');
+
+    expect(Link.offset).toEqual(1);
```

Comment:
The offset we get is only the line break after the code block, is that expected @atomrc ?